R/segmentation.R

```diff
@@ -288,8 +288,9 @@ setMethod(
   signature = c(x = 'Segmentation', y = 'SpatialPolygons'),
   definition = function(x, y, returnList = FALSE, fn = NULL, ...) {
     return(over(
-      x = as(object = x, Class = 'SpatialPolygons'),
-      y = as(object = y, Class = 'SpatialPolygons'),
+      x = as(object = x, Class = 'sf'),
+      y = as(object = y, Class = 'sf'),
+      sparse = F,
```

Comment:
Slight stylistic fix
```suggestion
      sparse = FALSE,
```